src/resdk/tables/qc.py

```diff
@@ -448,6 +469,24 @@ def _fetch_group(self, group_name: str) -> pd.DataFrame:
         ]
         data = [self._load_fetch(data_type) for data_type in data_types]
         df = pd.concat(data, axis=1)
+        return df
+
+    @lru_cache()
+    def _load_fetch(self, data_type):
+        df = super()._load_fetch(data_type)
```

Comment:
E2E tests are failing since you are not handling `_load_fetch(self.META)`. Add the following two lines after this one:
```
if data_type == self.META:
    return df
```